packages/components/src/FormField/Affix.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import { IconNames } from "@jobber/design";
+import styles from "./FormField.css";
+import { FormFieldProps } from "./FormFieldTypes";
+import { Icon } from "../Icon";
+
+export function Affix(props: FormFieldProps) {
```

Comment:
we tend to want to de-structure the props here to make the component/props more readable.